docs/maui/behaviors/touch-behavior.md

```diff
@@ -267,3 +267,94 @@ You can find an example of this behavior in action in the [.NET MAUI Community T
 ## API
 
 You can find the source code for `TouchBehavior` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Behaviors/TouchBehavior.shared.cs).
+
+
+## Migrating From Xamarin Community Toolkit
+
+In the [Xamarin Community Toolkit](https://github.com/xamarin/XamarinCommunityToolkit), there was the [`TouchEffect`](https://learn.microsoft.com/en-us/dotnet/api/xamarin.communitytoolkit.effects.toucheffect?view=xamarin-community-toolkit-sdk) if you are upgrading an app from Forms to Maui there are a number of changes that you need to be aware of.
```

Comment:
As we are on the Microsoft docs page we need to use the full names

```suggestion
In the [Xamarin Community Toolkit](https://github.com/xamarin/XamarinCommunityToolkit), there was the [`TouchEffect`](https://learn.microsoft.com/en-us/dotnet/api/xamarin.communitytoolkit.effects.toucheffect?view=xamarin-community-toolkit-sdk) if you are upgrading an app from Xamarin.Forms to .NET MAUI there are a number of changes that you need to be aware of.
```

Reply:
Fully qualified all names